### ISSUE TITLE
[question/sugestion] add linux for game

### ISSUE THREAD

**Issue by luckyrada** [tsunamayo/Starship-EVO#3384]
no idea if this is planned or has been sugested, but please make this game available for linux. i know it prob would be difficult... but it would be awesome.

**Comment by AlienXtream** [comment]
i cant speak for Tsuna but i would be surprised if linux support doesn't happen later down the line. at the moment however, with Tsuna being a solo dev, this just isn't practical. there are lots of changes being made quite often and that alone is a big workload. then taking into account different hardware and associated compatibility its already a monumental task. multi-platform support (i.e win/linux/mac) would just add another layer of complexity and that probably isn't a good idea right now. other linux users have had some success with the game however and it might be worth hopping over to the discord server and asking them how they did it and if its possible for you to do too. i know its not ideal but for now its the best i can suggest for linux users. 

tl:dr. Solo dev. Lots of work to do. multiplatform means MORE work. maybe later. ask discord server for workarounds.

hope that helps. if not then i apologise in advance :(

**Comment by luckyrada** [comment]
ok. thank you :)

**Comment by StarrKiss** [comment]
@luckyrada  I've been playing on Linux with Proton for MONTHS now. It's worked fine, just as well as on Windows, high framerates, the whole kebab. Just do that until he can release a Linux build later down the line.

**Comment by luckyrada** [comment]
great! i wondered if i could use it, as space engineers didn't work one bit no matter what i tried, i would like to ask though @StarrKiss , what OS are you using and what specs does your computer have?

**Comment by StarrKiss** [comment]
> great! i wondered if i could use it, as space engineers didn't work one bit no matter what i tried, i would like to ask though @StarrKiss , what OS are you using and what specs does your computer have?

Currently running Fedora KDE Spin, on my own PC with a GTX 1070 TI and Ryzen 7 3700x. But it also works on my brother's PC running Linux with a 1060 and a Ryzen 5, so take that how you will.

Operating System: Fedora Linux
KDE Plasma Version: 5.20.4
KDE Frameworks Version: 5.76.0
Qt Version: 5.15.2
Kernel Version: 5.9.11-3-FEDORA
OS Type: 64-bit
Processors: 16 × AMD Ryzen 7 3700X 8-Core Processor
Memory: 15.6 GiB of RAM
Graphics Processor: GeForce GTX 1070 Ti/PCIe/SSE2

**Comment by luckyrada** [comment]
ok, thank you so much

**Comment by tsunamayo** [comment]
@luckyrada @StarrKiss Thanks for info. Yes Linux port is not on the roadmap unfortunately, at least for a long long time. It is not worth my (extremely limited) time because of the small user base, I better off spend the time on make a survival mode for example.
Cheers

**Comment by luckyrada** [comment]
ok. i hope you could get around to it at some point :)

**Comment by mkaito** [comment]
For a one-man team, a native Linux port is just going to drive you nuts for little gain.

You could just test on Proton, maybe. You know, if you are inclined.

**Comment by najlitarvan** [comment]
ya, a proton support would be wondeful and imo would satisfy most people who want to play this game on linux
